vars/runIntegTestScript.groovy

```diff
@@ -0,0 +1,17 @@
+void call(Map args = [:]) {
+    String jobName = args.jobName ?: 'distribution-build-opensearch'
+    lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+    echo "args.buildManifest: ${args.buildManifest}"
+    echo "args.testManifest: ${args.testManifest}"
+    def buildManifest = lib.jenkins.BuildManifest.new(readYaml(file: args.buildManifest))
+    String artifactRootUrl = buildManifest.getArtifactRootUrl("${jobName}", "${BUILD_ID}")
+    echo "Artifact root URL: ${artifactRootUrl}"
+    
+    sh([
+        './test.sh',
+        'integ-test',
+        "${args.testManifest}",
+        "--paths opensearch=${artifactRootUrl}",
```

Comment:
I would prefer we were able to pass the manifest file directly to the python code instead of having to hack up the parameters.

Reply:
Keeping a single thread about this via https://github.com/opensearch-project/opensearch-build/pull/1429#discussion_r782485772